public/index.html

```diff
@@ -58,7 +58,7 @@
 
             <a-entity id="nametag" networked="template:#nametag-template;showLocalTemplate:false;"></a-entity>
 
-            <a-entity id="left-hand" hand-controls="left" axis-dpad="centerZone: 1" teleport-controls="cameraRig: #player-rig; teleportOrigin: #head; button: action_teleport_"
+            <a-entity id="left-hand" hand-controls="left" visible="false" hand-controls-child-visibility axis-dpad="centerZone: 1" teleport-controls="cameraRig: #player-rig; teleportOrigin: #head; button: action_teleport_"
```

Comment:
Might want to remove the `visible="false"` here since the `hand-controls-child-visibility` component sets this up in init. As it is now I might expect if I changed this to true that I would see them by default.

Reply:
Good point.